superface/super.json

```diff
@@ -1327,6 +1330,18 @@
         }
       ]
     },
+    "pipedrive": {
+      "file": "../providers/pipedrive.json",
+      "security": [
+        {
+          "id": "api-key",
+          "apikey": "$PIPEDRIVE_API_KEY"
+        }
+      ],
+      "parameters": {
+        "COMPANY_DOMAIN": "$PIPEDRIVE_COMPANY_DOMAIN"
```

Comment:
When you add a new environment variable, please update `.env.example` accordingly (same with `$PIPEDRIVE_API_KEY`)